client/components/settings/AccessTokenCard.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <div
+    class="text-gray-500 border shadow rounded-md p-5 mt-4 relative flex items-center"
+  >
+    <div class="flex-grow flex items-center">
+      <div>
+        <div class="flex space-x-3 font-semibold mr-2">
+          {{ token.name }}
+        </div>
+
+        <div>
+          {{ token.abilities }}
+        </div>
+      </div>
+    </div>
```

Comment:
![Created at](https://github.com/user-attachments/assets/1b20923d-8298-4f1c-b292-28942bee1193)
Can we add timestamps to the card? Whatever you think is best UI-wise

Reply:
Also, I don't think ability is displayed?